src/components/input/checkbox/CheckboxComponent.js

```diff
@@ -1,33 +1,51 @@
-import React, { useState, useEffect } from "react";
-import { Checkbox } from "@mui/material";
+import React, { useState, useEffect } from 'react';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import styled from 'styled-components';
+import { colors } from '../../colors/Colors';
 
-import { colors } from "../../colors/Colors";
+const Label = styled.div``;
```

Comment:
is this necessary compared to just putting `div`?

Reply:
Nope, I forgot to switch it to a regular div. Fixed.